frontend/src/app/plugins/shape.cljs

```diff
@@ -1018,7 +1018,7 @@
                  (u/display-not-valid :makeMask (:type shape))
 
                  :else
-                 (upf/format-path (:content shape)))))
+                 (str (:content shape)))))
```

Comment:
better to use `dm/str`?? in other places you've used `.toString` instead of plain `str`

Reply:
Just used str for cosmetic look and feel. I will revisit the usage of str and .toString for make it consistent on the code, maybe making a specific function for it just for syntactic convenience...

Reply:
Fixed